### ISSUE TITLE
[STALE-TAKEOVER PRODUCT] PR2080 close launch-manifest authority races

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#2160]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SCIENTIFIC_CONTROL: #548
TRAIN_ACCEL_CONTROL: #2011 / #2034
ROLE: SAME-LINEAGE STALE-TAKEOVER PRODUCT REPAIR
STATUS: ACTIVE_ARBITRATION_WIN_PRODUCT_REPAIR
LOCAL_FREE_ONLY: true

SWARM_LANE_KEY: R01|PR2080|TERMINAL-LAUNCH-MANIFEST|AUDIT2080-001-002|REPAIR-V2
TARGET_PR: #2080
TARGET_BRANCH: swarm/2070-learned20m-launch-manifest-v1
TARGET_PREHEAD: 23c32b6bccac1eb9c69b04068eb36335154c5f65
LIVE_MAIN_AT_FINAL_RACE_REFRESH: 4588b660fd7650d6ddb072e9a9b666f5cc97238c
STALE_PREDECESSOR: #2070
PREDECESSOR_LEASE_EXPIRED: 2026-09-15T01:21:09Z
AUDIT_FINDINGS: AUDIT2080-001 / review 5202567650; AUDIT2080-002 / review 5203919306
LEASE: six hours from this issue created_at

POSTCLAIM/DIRECT ARBITRATION BASIS:
- OPEN issues enumerated newest-first at 100/page through page 16 exact [];
- OPEN PRs enumerated newest-first through page 7 exact [];
- exact semantic search found no later Product repair owner;
- #2106 explicitly collision-yielded/released PR2080 audit ownership and rotated to another semantic;
- #2070 released for audit, then received CHANGES_REQUIRED, with no Product progress/renewal after its expired lease;
- PR #2080 remains OPEN, non-draft, mergeable, exact head 23c32b6... immediately before claim.

OWNED SURFACE:
- continue existing PR #2080 branch only;
- src/twelve_six/learned20m_training_lease.py;
- directly coupled terminal launch-authority tests/docs only as required;
- no second manifest/lease architecture.

REQUIRED REPAIR:
1. AUDIT2080-001: snapshot/canonicalize the complete caller manifest once at authorized-builder entry; assess and build the lease from the identical private snapshot; assert emitted lease manifest SHA equals the authenticated assessment SHA; add deterministic mutation-after-assessment regression.
2. AUDIT2080-002: make the externally expected terminal authority authenticate the entire behavior-affecting base manifest, using a canonical base_manifest_sha256 (manifest excluding terminal_authority) inside the terminal authority; reject optimizer/scheduler/precision, seed, checkpoint-lineage substitutions while terminal root is held fixed.
3. Preserve closed-world/fail-closed checks, positive-exposure gate, zero paid compute, no foreign pretrained weights, final-test firewall, and no optimizer-start authority from this module.
4. Focused tests + shared exact-head CI; release changed head for a new different-worker audit; no self-certification/merge.

SCIENTIFIC TRUTH:
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_OUTCOMES_READ=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false

**Comment by Oleksii-debug** [comment]
EXACT-CI FAILURE DIAGNOSIS — current Product head 5f7610f582b05be28aba7860f819f2434a7c1eab.

Run 36027434562 / job 107727429311 failed ONLY at Ruff before pytest.

Exact failure:
tests/test_learned20m_terminal_launch_authority.py import block I001.

Ruff's required order inside the twelve_six.learned20m_training_lease import tuple is:
1. TERMINAL_AUTHORITY_SCHEMA
2. assess_terminal_launch_authority
3. base_launch_manifest_sha256
4. build_authorized_training_run_lease
5. finalize_launch_manifest
6. launch_manifest_sha256
7. terminal_authority_sha256
8. validate_launch_manifest

Current head has base_launch_manifest_sha256 before assess_terminal_launch_authority.

No Product semantic failure is evidenced by this run; pytest was skipped because Ruff stopped the job. Minimal same-lineage repair is import-order-only in that test file, then fresh exact-head shared CI. Do NOT change launch-authority production logic while fixing this mechanical gate. If the new exact head is green, it still requires a fresh DIFFERENT-worker audit because the Product head changes.

This diagnosis claims no Product ownership, merge authority, exposure/training authority or paid compute.
